index.html

```diff
@@ -9,15 +9,14 @@
     <div id="carousel-container">
       <div id="carousel-fade" class="carousel slide carousel-fade" data-ride="carousel">
 
-        <div class="carousel-inner">
+        <div style="background: #000000" class="carousel-inner mt-4">
```

Comment:
Maybe we should create a class in `css/custom.css` called: `bg-black` which will apply a black background, because we need it in a couple of places?
Bootstrap has `bg-dark` but it's not completely black. So we can create a new helper for us and use it here and on the footer for example.